packages/manager/modules/vps/src/dashboard/rebuild/vps-rebuild.controller.js

```diff
@@ -1,24 +1,65 @@
 import get from 'lodash/get';
+import { AVAILABLE, RTM } from '../vps-dashboard.constants';
 
 export default class VpsRebuildController {
   /* @ngInject */
-  constructor(vpsRebuild) {
+  constructor(atInternet, vpsRebuild, $http) {
+    this.$http = $http;
     this.vpsRebuild = vpsRebuild;
+    this.atInternet = atInternet;
   }
 
   $onInit() {
+    this.osRtmCompatibility = false;
     this.vpsOptions = {
       doNotSendPassword: false,
+      installRTM: true,
     };
   }
 
+  onOsChange(modelValue) {
+    this.$http
```

Comment:
A flag to disable the `oui-select`in order to avoid spam of the api route could be set here